src/main/kotlin/it/pagopa/helpdeskcommands/controllers/CommandsController.kt

```diff
@@ -18,14 +18,16 @@ class CommandsController(@Autowired private val commandsService: CommandsService
 
     override fun refundOperation(
         xUserId: String,
+        xForwardedFor: String?,
```

Comment:
```suggestion
        xForwardedFor: String,
```
since it's declared as required in openapi definition can be defined as non-nullable string here :D

Reply:
sure, thanks